src/components/UserLogin/Login.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import "./Login.css";
+
+const Login = () => {
+	const [email, setEmail] = useState("");
+	const [password, setPassword] = useState("");
+	const [error, setError] = useState("");
+	const [emailError, setEmailError] = useState("");
+	const [showPassword, setShowPassword] = useState(false);
+	const navigate = useNavigate();
+
+	const validateEmail = (email) => {
+		const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+		return emailRegex.test(email);
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		setError("");
+		setEmailError("");
+		if (!validateEmail(email)) {
+			setEmailError("Invalid email format");
+			return;
+		}
+
+		try {
+			const response = await fetch(
+				"http://localhost:8000/api/v1/auth/login",
```

Comment:
Same comment I added to Beth's PR, let's not hard-code a URL in here for the backend. Instead, let's load it as an environment variable from the `.env` file so it's easy to swap between the production and a locally-running instance of the backend.

i.e. `process.meta.env + "/api/v1/auth/login"`

https://vite.dev/guide/env-and-mode